src/nvim/ex_cmds.c

```diff
@@ -5076,15 +5098,15 @@ int find_help_tags(const char *arg, int *num_matches, char ***matches, bool keep
   } else {
     // Recognize a few exceptions to the rule.  Some strings that contain
     // '*' with "star".  Otherwise '*' is recognized as a wildcard.
```

Comment:
Missing comment change
```suggestion
    // '*'are changed to "star", otherwise '*' is recognized as a wildcard.
```